qa/suites/orch/cephadm/thrash/1-start.yaml

```diff
@@ -1,3 +1,8 @@
+overrides:
+  ceph:
+    log-ignorelist:
+      - MON_DOWN
+      - OSDMAP_FLAGS
```

Comment:
thrash tests also need to ignore `PG_DEGRADED`, `OSD_DOWN`, and `POOL_APP_NOT_ENABLED`
```
"2024-02-27T07:50:00.000142+0000 mon.a (mon.0) 896 : cluster 3 [WRN] PG_DEGRADED: Degraded data redundancy: 3284/201426 objects degraded (1.630%), 2 pgs degraded" in cluster log
```
```
"2024-02-27T08:20:00.000210+0000 mon.a (mon.0) 1841 : cluster 3 [WRN] OSD_DOWN: 1 osds down" in cluster log
```
```
"2024-02-27T06:40:00.000211+0000 mon.a (mon.0) 1620 : cluster 3 [WRN] POOL_APP_NOT_ENABLED: 1 pool(s) do not have an application enabled" in cluster log
```

Reply:
one thrash test still failing for  `CACHE_POOL_NO_HIT_SET` warning, so will need that as well